### PR TITLE
Modified the activity feed 

### PR DESCRIPTION
**Pull request by VillanuevaK**
Modified the activity feed servlet so that it could access conversations and modified the jsp file so that it loops through all the conversations and shows them

### CODE REVIEW

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 1]
```diff
@@ -1,27 +1,83 @@
...
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+import java.util.UUID;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/*Servlet class responsible for the activity feed page*/
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: nitpick:
Add space before and after the comment sentence. Also period at the end of the sentence.
/* Servlet class responsible for the activity feed page. */

Please apply the same change to comments throughout the file .

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 2]
```diff
@@ -1,27 +1,83 @@
...
 public class ActivityServlet extends HttpServlet {
+        
+    /*Store class that gives access to users*/
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: nit:
Align comment with the line 'private UserStore userStore;' below

Similarly to the comment below.

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 3]
```diff
@@ -1,27 +1,83 @@
...
+	private UserStore userStore;
+    /*Store class that gives access to conversations*/
+	private ConversationStore conversationStore;
+	/** Store class that gives access to Messages. */
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: Here you have '/** ' whereas above you use '/* '
Here are the difference https://stackoverflow.com/questions/29815636/and-in-java-comments/47787786

Prefer '/** ' to generate java doc.

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 4]
```diff
@@ -1,27 +1,83 @@
...
+	private MessageStore messageStore;
+
+
+	/**
+	* Set up state for handling conversation-related requests. This method is only called when
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: nit:
Misalignment of comment.

The asterisk at the beginning of '* Setup' should align with the second asterisk from the first line '/**'

Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: nit:
Update comment
'handling conversation-related requests' -> 'handling activity-related requests'

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 5]
```diff
@@ -1,27 +1,83 @@
...
+	* running in a server, not when running in a test.
+	*/
+	@Override
+	public void init() throws ServletException {
+		super.init();
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: 'super.init()' should align with 'b' in 'public void init()'
Look at exiting code for reference.
Also https://google.github.io/styleguide/javaguide.html

File: src/main/java/codeu/controller/ActivityServlet.java [hunk 6]
```diff
@@ -1,27 +1,83 @@
...
+
+		/*Get all convos*/
+		List<Conversation> conversations = conversationStore.getAllConversations();
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response)
-      throws IOException, ServletException {
-    request.getRequestDispatcher("/WEB-INF/view/activity.jsp").forward(request, response);
-  }
+		/*make the conversations variable accesible to the jsp file*/
```
Comment (yufangqq) [inline src/main/java/codeu/controller/ActivityServlet.java]: Again, comment format issue.
Please take a look at https://github.com/google/google-java-format and use it to reformat your code.
And see what changes dose the tool make.

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 7]
```diff
@@ -7,6 +17,25 @@
 </head>
```
Comment (yufangqq) [inline src/main/webapp/WEB-INF/view/activity.jsp:16]: I think you forgot to delete the </head> and <body> here.

You added a new </head> and <body> below in this pull request.

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 8]
```diff
@@ -19,6 +48,19 @@
     <a href="/activity.jsp">Activity Feed</a>
   </nav>
 
+<div id="feed">
+  <ul>
+    <%
+      for (Conversation convo : conversations) {
```
Comment (yufangqq) [inline src/main/webapp/WEB-INF/view/activity.jsp]: nit:

(Conversation conversation : conversations)

'convo' is not a well-known abbreviation for conversation (as far as I know), so naming that various 'conversation' is better, more clear, more readable.

File: src/main/webapp/WEB-INF/view/activity.jsp [hunk 9]
```diff
@@ -19,6 +48,19 @@
...
+    <li><strong><%= title %></strong></li>
+    <%
+      }
+    %>
+  </ul>
+</div>
+
   <div id="container">
 
     <% if(request.getAttribute("error") != null){ %>
```
Comment (yufangqq) [inline src/main/webapp/WEB-INF/view/activity.jsp:65]: ??? What is going on here?
is this
<% if(request.getAttribute("error") != null){ %>
supposed to be the last line of the file?

Where is the rest of the file?

Comment (VillanuevaK) [reply src/main/webapp/WEB-INF/view/activity.jsp:65]: I've fixed the style issues, but I will further review the guide when I have more time. As far as the "missing" part of the file, I have no idea. The code I committed is complete so once I commit again with the new updates hopefully the entire file will show.

Comment (VillanuevaK) [reply src/main/webapp/WEB-INF/view/activity.jsp:65]: Actually I believe that it only shows the parts of the files that have been changed. The rest is still there (Can be seen on the master branch).

**Pull request merged**